modules/insights/private-link-scope/main.bicep

```diff
@@ -6,6 +6,9 @@ metadata owner = 'Azure/module-maintainers'
 @minLength(1)
 param name string
 
+@description('Required. Specifies the access mode of ingestion or queries through associated private endpoints in scope.')
```

Comment:
Please run the `utilities/tools/Set-Module` utility on the module folder to have it update both the `main.json` & `readme.md` files

Reply:
Done, will update when other changes are required.